tfjs-core/package.json

```diff
@@ -78,7 +78,8 @@
     "@types/webgl-ext": "0.0.30",
     "long": "4.0.0",
     "node-fetch": "~2.6.1",
-    "seedrandom": "2.4.3"
+    "seedrandom": "2.4.3",
+    "@webgpu/types": "0.1.6"
```

Comment:
We need to use "@webgpu/types" defined in tfjs/package.json instead.

Reply:
tfjs/package.json had already added this package, so I removed it here.